pages/view/_iscnId.vue

```diff
@@ -191,41 +395,111 @@ export default class ViewIscnIdPage extends Vue {
     return this.record.stakeholders
   }
 
-  created() {
-    const { iscnId } = this.$route.params
-    if (iscnId.startsWith(ISCN_PREFIX)) {
-      this.iscnId = iscnId
-    }
+  get transactionsURL() {
+    return `${BIG_DIPPER_TX_BASE_URL}${this.txHash}`
   }
 
-  async mounted() {
-    if (!this.iscnId) {
-      const param = this.$route.params.iscnId
-      if (!isCosmosTransactionHash(param)) {
-        this.$nuxt.error({
-          statusCode: 400,
-          message: ErrorMessage.statusCode400,
+  get rawDataURL() {
+    return `${ISCN_RAW_DATA_ENDPOINT}${this.iscnId}`
+  }
+
+  async fetchTxHash() {
+    const res = await this.$axios.get(`${ISCN_TX_RAW_DATA_ENDPOINT}${this.iscnId}`)
+    const txHash = this.getTxHash(res.data.txs)
+    return txHash
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  getTxHash(items: any) {
+      const targetItem = items.find((item: any) =>
+        item.logs.find((log: any) =>
+          log.events.find((event: any) =>
+            event.attributes.some(
+              (attribute: any) =>
+                attribute.key === 'action' &&
+                attribute.value === 'create_iscn_record',
+            ),
+          ),
+        ),
+      )
+      return targetItem.txhash
+    }
+
+  showStakeholder(index: number) {
+    this.isOpenAuthorDialog = true
+    const stakeholders = this.stakeholders[index].entity
+    
+    if (this.stakeholders[index].entity.identifier) {
+      const { description: authorDescription, name: authorName } = stakeholders
+      const likerId = stakeholders!.url.includes('like.co')
+        ? stakeholders.url.slice(16)
+        : ''
+      const authorWalletAddresses: any = []
+      stakeholders.identifier.forEach((a: any) => {
+        authorWalletAddresses.push({
+          address: a.value,
+          type: this.getKeyByValue(WALLET_TYPE_REPLACER, a.propertyID),
         })
-        return
+      })
```

Comment:
Since its onc-to-one array mapping, you can use `map()`
```
const authorWalletAddresses = stakeholders.identifier.map((a: any) => {
        return {
          address: a.value,
          type: this.getKeyByValue(WALLET_TYPE_REPLACER, a.propertyID),
        }
      })
```